### ISSUE TITLE
please help

### ISSUE THREAD

**Issue by Ampdale** [ahmetcandan/WebHost#5]
https://github.com/ahmetcandan/Chat/issues/3

**Comment by ahmetcandan** [comment]
Of course, this process can be done by keeping user information in a database on the server side. 
I migrated the application to .Net 8.0.
I will add it when I finish my development about it.
You should not need to take any action for public IP usage. If your IP and port number is open, it should work.
